downstreamadapter/dispatchermanager/event_dispatcher_manager.go

```diff
@@ -106,6 +113,11 @@ type EventDispatcherManager struct {
 
 	// sink is used to send all the events to the downstream.
 	sink sink.Sink
+	// redo related
+	RedoEnable bool
```

Comment:
use `redoEnable` instead of `redoEnable`

Reply:
It will be used by `dispatcherorchestrator`